instruqt/proxy/vendor/manifests/nginx-service.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+  annotations:
+    kots.io/when: '{{repl not IsKurl }}'
+spec:
+  type: ClusterIP
+  ports:
+  - port: 80
+  selector:
+    app: nginx
```

Comment:
Well maybe, but the code might be lifted by a vendor and is it not good practice to have these annotations for when vendors allow their app to be deployed in both embedded cluster and existing cluster scenarios?  Agreed that in the context of this lab though it will always be kurl :p